_posts/2020-07-19-stage-set-for-knockouts.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: news
+title: The stage is set for the SR2020 Knockouts
+---
+
+After two weekends and 48 matches, the league stage of the SR2020 Virtual
+Competition is complete, setting the stage for the knockouts and the grand final
+next weekend.
+
+This weekend saw strong performances from a number of teams, including a number
+of firsts: Kenilworth Sixth Form collected 5 tokens into their scoring zone in
+one match and both Hills Road Sixth Form College's first team and JAMDynamics
+stealing tokens from each others scoring zones.
```

Comment:
If the livestream is ready soon, might be nice to link to the specific timings of these events.
Not a blocker though -- it's a small thing and we can add it later if we want to .